### ISSUE TITLE
Exporter SystemErrors query returns duplicates (breaks all metrics)

### ISSUE THREAD

**Issue by gmarkey** [Altinity/clickhouse-operator#1161]
Using image `docker.io/altinity/metrics-exporter:0.21.0` as part of the operator (0.21.0). Clickhouse version 22.8.16.32.

After our cluster had some (unrelated) issues, the exporter completely stopped working. Below shows the output of the `/metrics` endpoint:

```
bash-5.2$ curl http://localhost:8888/metrics
An error has occurred while serving metrics:

7 error(s) occurred:
* collected metric "chi_clickhouse_metric_SystemErrors_TOO_MANY_SIMULTANEOUS_QUERIES" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s2-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:218 > } was collected before with the same name and label values
* collected metric "chi_clickhouse_metric_SystemErrors_CANNOT_SCHEDULE_TASK" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s2-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:26 > } was collected before with the same name and label values
* collected metric "chi_clickhouse_metric_SystemErrors_TOO_MANY_SIMULTANEOUS_QUERIES" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s1-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:323 > } was collected before with the same name and label values
* collected metric "chi_clickhouse_metric_SystemErrors_QUERY_WAS_CANCELLED" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s1-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:1 > } was collected before with the same name and label values
* collected metric "chi_clickhouse_metric_SystemErrors_CANNOT_SCHEDULE_TASK" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s1-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:18 > } was collected before with the same name and label values
* collected metric "chi_clickhouse_metric_SystemErrors_QUERY_WAS_CANCELLED" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s0-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:1 > } was collected before with the same name and label values
* collected metric "chi_clickhouse_metric_SystemErrors_CANNOT_SCHEDULE_TASK" { label:<name:"chi" value:"clickhouse-prod" > label:<name:"hostname" value:"sh-0r-s0-r0.clickhouse-prod.svc.cluster.local" > label:<name:"namespace" value:"clickhouse-prod" > gauge:<value:118 > } was collected before with the same name and label values
```

When I run the query against a server I get duplicate rows:
```
SELECT
    concat('metric.SystemErrors_', errorCodeToName(code)) AS metric,
    toString(value) AS value,
    'Error counter from system.errors' AS description,
    'gauge' AS type
FROM system.errors

Query id: c58c608f-0eb1-4287-8f78-2ed139a146e0

┌─metric────────────────────────────────────────────┬─value─┬─description──────────────────────┬─type──┐
│ metric.SystemErrors_CANNOT_WRITE_TO_OSTREAM       │ 855   │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_ILLEGAL_TYPE_OF_ARGUMENT      │ 2     │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_UNKNOWN_FUNCTION              │ 2     │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_UNKNOWN_IDENTIFIER            │ 12    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_SYNTAX_ERROR                  │ 53    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_UNKNOWN_PACKET_FROM_CLIENT    │ 1     │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_UNKNOWN_SETTING               │ 2     │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_NO_ELEMENTS_IN_CONFIG         │ 39    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_IP_ADDRESS_NOT_ALLOWED        │ 39    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_TOO_MANY_SIMULTANEOUS_QUERIES │ 377   │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_TOO_MANY_SIMULTANEOUS_QUERIES │ 323   │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_SOCKET_TIMEOUT                │ 93    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_NETWORK_ERROR                 │ 6341  │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_MEMORY_LIMIT_EXCEEDED         │ 58    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_QUERY_WAS_CANCELLED           │ 1     │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_QUERY_WAS_CANCELLED           │ 1     │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_CANNOT_SCHEDULE_TASK          │ 1240  │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_CANNOT_SCHEDULE_TASK          │ 18    │ Error counter from system.errors │ gauge │
│ metric.SystemErrors_AUTHENTICATION_FAILED         │ 39    │ Error counter from system.errors │ gauge │
└───────────────────────────────────────────────────┴───────┴──────────────────────────────────┴───────┘
```

Perhaps it needs something like the following:
```
SELECT
    concat('metric.SystemErrors_', errorCodeToName(code)) AS metric,
    toString(sum(value)) AS value,
    'Error counter from system.errors' AS description,
    'gauge' AS type
FROM system.errors
GROUP BY metric
```

**Comment by Slach** [comment]
Thanks for reporting!
Could you share the results for the following queue?

```sql 
SELECT name, code, errorCodeToName(code) AS metric, value FROM system.errors ORDER BY metric;
```

**Comment by gmarkey** [comment]
> Thanks for reporting! Could you share the results for the following queue?
> 
> ```sql
> SELECT name, code, errorCodeToName(code) AS metric, value FROM system.errors ORDER BY metric;
> ```

Hey @Slach no problem. Here are the results:
```
SELECT
    name,
    code,
    errorCodeToName(code) AS metric,
    value
FROM system.errors
ORDER BY metric ASC

Query id: 864e7c56-0986-46c7-b434-c42b5b2f6ca3

┌─name──────────────────────────┬─code─┬─metric────────────────────────┬─value─┐
│ AUTHENTICATION_FAILED         │  516 │ AUTHENTICATION_FAILED         │   191 │
│ CANNOT_SCHEDULE_TASK          │  439 │ CANNOT_SCHEDULE_TASK          │  1240 │
│ CANNOT_SCHEDULE_TASK          │  439 │ CANNOT_SCHEDULE_TASK          │    18 │
│ CANNOT_WRITE_TO_OSTREAM       │   24 │ CANNOT_WRITE_TO_OSTREAM       │   863 │
│ ILLEGAL_TYPE_OF_ARGUMENT      │   43 │ ILLEGAL_TYPE_OF_ARGUMENT      │     2 │
│ IP_ADDRESS_NOT_ALLOWED        │  195 │ IP_ADDRESS_NOT_ALLOWED        │   191 │
│ MEMORY_LIMIT_EXCEEDED         │  241 │ MEMORY_LIMIT_EXCEEDED         │    58 │
│ NETWORK_ERROR                 │  210 │ NETWORK_ERROR                 │  9996 │
│ NOT_IMPLEMENTED               │   48 │ NOT_IMPLEMENTED               │     1 │
│ NO_ELEMENTS_IN_CONFIG         │  139 │ NO_ELEMENTS_IN_CONFIG         │    59 │
│ QUERY_WAS_CANCELLED           │  394 │ QUERY_WAS_CANCELLED           │     1 │
│ QUERY_WAS_CANCELLED           │  394 │ QUERY_WAS_CANCELLED           │     1 │
│ SOCKET_TIMEOUT                │  209 │ SOCKET_TIMEOUT                │    94 │
│ SYNTAX_ERROR                  │   62 │ SYNTAX_ERROR                  │    70 │
│ TOO_MANY_SIMULTANEOUS_QUERIES │  202 │ TOO_MANY_SIMULTANEOUS_QUERIES │   377 │
│ TOO_MANY_SIMULTANEOUS_QUERIES │  202 │ TOO_MANY_SIMULTANEOUS_QUERIES │   323 │
│ UNKNOWN_FUNCTION              │   46 │ UNKNOWN_FUNCTION              │     2 │
│ UNKNOWN_IDENTIFIER            │   47 │ UNKNOWN_IDENTIFIER            │    18 │
│ UNKNOWN_PACKET_FROM_CLIENT    │   99 │ UNKNOWN_PACKET_FROM_CLIENT    │     1 │
│ UNKNOWN_SETTING               │  115 │ UNKNOWN_SETTING               │     2 │
│ UNKNOWN_TABLE                 │   60 │ UNKNOWN_TABLE                 │     8 │
└───────────────────────────────┴──────┴───────────────────────────────┴───────┘
```

**Comment by Slach** [comment]
hm, ooks weird
lets check `last_error_message`
```sql
SELECT name, code, errorCodeToName(code) AS metric, value, last_error_message FROM system.errors ORDER BY metric;
```

**Comment by gmarkey** [comment]
> hm, looks weird lets check `last_error_message`
> 
> ```sql
> SELECT name, code, errorCodeToName(code) AS metric, value, last_error_message FROM system.errors ORDER BY metric;
> ```
┌─[1mname[0m──────────────────────────┬─[1mcode[0m─┬─[1mmetric[0m────────────────────────┬─[1mvalue[0m─┬─[1mlast_error_message[0m─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ ALL_CONNECTION_TRIES_FAILED   │  279 │ ALL_CONNECTION_TRIES_FAILED   │  1679 │ All connection tries failed. Log: 

Code: 210. DB::NetException: I/O error: Broken pipe, while writing to socket (x.x.x.x:9000). (NETWORK_ERROR) (version 22.8.16.32 (official build))
Code: 210. DB::NetException: I/O error: Broken pipe, while writing to socket (x.x.x.x:9000). (NETWORK_ERROR) (version 22.8.16.32 (official build))
Timeout exceeded while reading from socket (sh-0r-s1-r0:9000, receive timeout 1800000 ms)

 │
│ ATTEMPT_TO_READ_AFTER_EOF     │   32 │ ATTEMPT_TO_READ_AFTER_EOF     │    10 │ Attempt to read after eof                                                                                                                                                                                                                                  │
│ AUTHENTICATION_FAILED         │  516 │ AUTHENTICATION_FAILED         │   151 │ clickhouse_operator: Authentication failed: password is incorrect or there is no user with such name                                                                                                                                                       │
│ BAD_ARGUMENTS                 │   36 │ BAD_ARGUMENTS                 │     5 │ HR doesn't look like datepart name in date_trunc                                                                                                                                                                                                           │
│ BAD_GET                       │  170 │ BAD_GET                       │     1 │ Requested cluster 'default' not found                                                                                                                                                                                                                      │
│ CANNOT_PARSE_DATETIME         │   41 │ CANNOT_PARSE_DATETIME         │     1 │ Cannot parse datetime 2023-05-26 14:00                                                                                                                                                                                                                     │
│ CANNOT_SCHEDULE_TASK          │  439 │ CANNOT_SCHEDULE_TASK          │  1098 │ Cannot schedule a task: no free thread (timeout=0) (threads=10000, jobs=10000)                                                                                                                                                                             │
│ CANNOT_SCHEDULE_TASK          │  439 │ CANNOT_SCHEDULE_TASK          │    26 │ Received from sh-0r-s0-r0:9000. DB::Exception: Cannot schedule a task: no free thread (timeout=0) (threads=10000, jobs=10000). Stack trace:

0. DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int, bool) @ 0x8e2bb88 in /usr/bin/clickhouse
1. DB::Exception::Exception<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, unsigned long, unsigned long&>(int, fmt::v8::basic_format_string<char, fmt::v8::type_identity<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&>::type, fmt::v8::type_identity<unsigned long>::type, fmt::v8::type_identity<unsigned long&>::type>, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, unsigned long&&, unsigned long&) @ 0x8ee12f4 in /usr/bin/clickhouse
2. void ThreadPoolImpl<std::__1::thread>::scheduleImpl<void>(std::__1::function<void ()>, int, std::__1::optional<unsigned long>)::'lambda'(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)::operator()(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) const @ 0x8ee0fe0 in /usr/bin/clickhouse
3. void ThreadPoolImpl<std::__1::thread>::scheduleImpl<void>(std::__1::function<void ()>, int, std::__1::optional<unsigned long>) @ 0x8edcbdc in /usr/bin/clickhouse
4. ? @ 0x128a1cb4 in /usr/bin/clickhouse
5. DB::PipelineExecutor::spawnThreads() @ 0x128a0184 in /usr/bin/clickhouse
6. DB::PipelineExecutor::executeImpl(unsigned long) @ 0x1289e62c in /usr/bin/clickhouse
7. DB::PipelineExecutor::execute(unsigned long) @ 0x1289e434 in /usr/bin/clickhouse
8. ? @ 0x128ad7ac in /usr/bin/clickhouse
9. ThreadPoolImpl<std::__1::thread>::worker(std::__1::__list_iterator<std::__1::thread, void*>) @ 0x8ede4b0 in /usr/bin/clickhouse
10. ? @ 0x8ee13a4 in /usr/bin/clickhouse
11. start_thread @ 0x7624 in /usr/lib/aarch64-linux-gnu/libpthread-2.31.so
12. ? @ 0xd149c in /usr/lib/aarch64-linux-gnu/libc-2.31.so
 │
│ CANNOT_WRITE_TO_OSTREAM       │   24 │ CANNOT_WRITE_TO_OSTREAM       │  1112 │ Cannot write to ostream at offset 402660                                                                                                                                                                                                                   │
│ ILLEGAL_TYPE_OF_ARGUMENT      │   43 │ ILLEGAL_TYPE_OF_ARGUMENT      │     3 │ Illegal types of arguments: Map(String, UInt8), Enum8('Green' = 1, 'Yellow' = 2, 'Orange' = 3, 'Red' = 4) for function arrayElement                                                                                                                        │
│ IP_ADDRESS_NOT_ALLOWED        │  195 │ IP_ADDRESS_NOT_ALLOWED        │   151 │ Connections from ::ffff:x.x.x.x are not allowed                                                                                                                                                                                                      │
│ MEMORY_LIMIT_EXCEEDED         │  241 │ MEMORY_LIMIT_EXCEEDED         │     2 │ Received from sh-0r-s1-r0:9000. DB::Exception: Memory limit (for query) exceeded: would use 9.45 GiB (attempt to allocate chunk of 5037611 bytes), maximum: 9.31 GiB. OvercommitTracker decision: Memory overcommit isn't used. OvercommitTracker isn't set.: (while reading column query): (while reading from part /var/lib/clickhouse/store/227/227201cd-4696-452b-bea6-45caf8b9d6d9/20230525_730744_734288_15/ from mark 0 with max_rows_to_read = 4842): While executing MergeTreeThread. Stack trace:

0. DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int, bool) @ 0x8e2bb88 in /usr/bin/clickhouse
1. DB::Exception::Exception<char const*, char const*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, long&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::basic_string_view<char, std::__1::char_traits<char> > >(int, fmt::v8::basic_format_string<char, fmt::v8::type_identity<char const*>::type, fmt::v8::type_identity<char const*>::type, fmt::v8::type_identity<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >::type, fmt::v8::type_identity<long&>::type, fmt::v8::type_identity<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >::type, fmt::v8::type_identity<std::__1::basic_string_view<char, std::__1::char_traits<char> > >::type>, char const*&&, char const*&&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&&, long&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&&, std::__1::basic_string_view<char, std::__1::char_traits<char> >&&) @ 0x8e1e628 in /usr/bin/clickhouse
2. MemoryTracker::allocImpl(long, bool, MemoryTracker*) @ 0x8e1def4 in /usr/bin/clickhouse
3. MemoryTracker::allocImpl(long, bool, MemoryTracker*) @ 0x8e1d910 in /usr/bin/clickhouse
4. CurrentMemoryTracker::realloc(long, long) @ 0x8e1bf40 in /usr/bin/clickhouse
5. Allocator<false, false>::realloc(void*, unsigned long, unsigned long, unsigned long) @ 0x8c3a960 in /usr/bin/clickhouse
6. ? @ 0x111c17e4 in /usr/bin/clickhouse
7. DB::ISerialization::deserializeBinaryBulkWithMultipleStreams(COW<DB::IColumn>::immutable_ptr<DB::IColumn>&, unsigned long, DB::ISerialization::DeserializeBinaryBulkSettings&, std::__1::shared_ptr<DB::ISerialization::DeserializeBinaryBulkState>&, std::__1::unordered_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, COW<DB::IColumn>::immutable_ptr<DB::IColumn>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const, COW<DB::IColumn>::immutable_ptr<DB::IColumn> > > >*) const @ 0x11182db0 in /usr/bin/clickhouse
8. DB::MergeTreeReaderWide::readData(DB::NameAndTypePair const&, std::__1::shared_ptr<DB::ISerialization const> const&, COW<DB::IColumn>::immutable_ptr<DB::IColumn>&, unsigned long, bool, unsigned long, unsigned long, std::__1::unordered_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, COW<DB::IColumn>::immutable_ptr<DB::IColumn>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const, COW<DB::IColumn>::immutable_ptr<DB::IColumn> > > >&, bool) @ 0x125dad4c in /usr/bin/clickhouse
9. DB::MergeTreeReaderWide::readRows(unsigned long, unsigned long, bool, unsigned long, std::__1::vector<COW<DB::IColumn>::immutable_ptr<DB::IColumn>, std::__1::allocator<COW<DB::IColumn>::immutable_ptr<DB::IColumn> > >&) @ 0x125da254 in /usr/bin/clickhouse
10. DB::MergeTreeRangeReader::DelayedStream::finalize(std::__1::vector<COW<DB::IColumn>::immutable_ptr<DB::IColumn>, std::__1::allocator<COW<DB::IColumn>::immutable_ptr<DB::IColumn> > >&) @ 0x12bcec7c in /usr/bin/clickhouse
11. DB::MergeTreeRangeReader::startReadingChain(unsigned long, std::__1::deque<DB::MarkRange, std::__1::allocator<DB::MarkRange> >&) @ 0x12bd83f0 in /usr/bin/clickhouse
12. DB::MergeTreeRangeReader::read(unsigned long, std::__1::deque<DB::MarkRange, std::__1::allocator<DB::MarkRange> >&) @ 0x12bd627c in /usr/bin/clickhouse
13. DB::MergeTreeBaseSelectProcessor::readFromPartImpl() @ 0x12bc8758 in /usr/bin/clickhouse
14. DB::MergeTreeBaseSelectProcessor::generate() @ 0x12bc4f68 in /usr/bin/clickhouse
15. DB::ISource::tryGenerate() @ 0x1289024c in /usr/bin/clickhouse
16. DB::ISource::work() @ 0x1288fd30 in /usr/bin/clickhouse
17. DB::ExecutionThreadContext::executeTask() @ 0x128a9f54 in /usr/bin/clickhouse
18. DB::PipelineExecutor::executeStepImpl(unsigned long, std::__1::atomic<bool>*) @ 0x1289efcc in /usr/bin/clickhouse
19. ? @ 0x128a1fd8 in /usr/bin/clickhouse
20. ThreadPoolImpl<std::__1::thread>::worker(std::__1::__list_iterator<std::__1::thread, void*>) @ 0x8ede4b0 in /usr/bin/clickhouse
21. ? @ 0x8ee13a4 in /usr/bin/clickhouse
22. start_thread @ 0x7624 in /usr/lib/aarch64-linux-gnu/libpthread-2.31.so
23. ? @ 0xd149c in /usr/lib/aarch64-linux-gnu/libc-2.31.so
 │
│ NETWORK_ERROR                 │  210 │ NETWORK_ERROR                 │ 11685 │ Connection reset by peer, while reading from socket ([::ffff:x.x.x.x]:48166)                                                                                                                                                                         │
│ NOT_AN_AGGREGATE              │  215 │ NOT_AN_AGGREGATE              │     3 │ Column `exchange` is not under aggregate function and not in GROUP BY. Have columns: ['max(toInt64(colour))']                                                                                                                                              │
│ NOT_IMPLEMENTED               │   48 │ NOT_IMPLEMENTED               │     1 │ Truncate is not supported by storage SystemErrors                                                                                                                                                                                                          │
│ NO_ELEMENTS_IN_CONFIG         │  139 │ NO_ELEMENTS_IN_CONFIG         │    75 │ Volume must contain at least one disk                                                                                                                                                                                                                      │
│ NO_REMOTE_SHARD_AVAILABLE     │  519 │ NO_REMOTE_SHARD_AVAILABLE     │     4 │ All attempts to get table structure failed. Log: 


                                                                                                                                                                                                          │
│ QUERY_WAS_CANCELLED           │  394 │ QUERY_WAS_CANCELLED           │     3 │ Query was cancelled                                                                                                                                                                                                                                        │
│ SOCKET_TIMEOUT                │  209 │ SOCKET_TIMEOUT                │  6508 │ Read timeout (3000 ms) while draining from epoll                                                                                                                                                                                                           │
│ SYNTAX_ERROR                  │   62 │ SYNTAX_ERROR                  │   375 │ Syntax error: failed at position 271 ('process_name') (line 10, col 22): process_name

GROUP BY process_name, t

ORDER BY process_name, t FORMAT JSON
. Expected one of: token, DoubleColon, MOD, DIV, NOT, BETWEEN, LIKE, ILIKE, NOT LIKE, NOT ILIKE, IN, NOT IN, GLOBAL IN, GLOBAL NOT IN, IS, AND, OR, QuestionMark, alias, AS, GROUP BY, WITH, HAVING, WINDOW, ORDER BY, LIMIT, OFFSET, SETTINGS, UNION, EXCEPT, INTERSECT, INTO OUTFILE, FORMAT, end of query │
│ TOO_MANY_SIMULTANEOUS_QUERIES │  202 │ TOO_MANY_SIMULTANEOUS_QUERIES │  1508 │ Too many simultaneous queries. Maximum: 800                                                                                                                                                                                                                │
│ TOO_MANY_SIMULTANEOUS_QUERIES │  202 │ TOO_MANY_SIMULTANEOUS_QUERIES │   218 │ Received from sh-0r-s1-r0:9000. DB::Exception: Too many simultaneous queries. Maximum: 800. Stack trace:

0. DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int, bool) @ 0x8e2bb88 in /usr/bin/clickhouse
1. DB::ProcessList::insert(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, DB::IAST const*, std::__1::shared_ptr<DB::Context const>) @ 0x11c6f454 in /usr/bin/clickhouse
2. ? @ 0x11ea65b8 in /usr/bin/clickhouse
3. DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::shared_ptr<DB::Context>, bool, DB::QueryProcessingStage::Enum) @ 0x11ea52fc in /usr/bin/clickhouse
4. DB::TCPHandler::runImpl() @ 0x12843d84 in /usr/bin/clickhouse
5. DB::TCPHandler::run() @ 0x12855f64 in /usr/bin/clickhouse
6. Poco::Net::TCPServerConnection::start() @ 0x13477f40 in /usr/bin/clickhouse
7. Poco::Net::TCPServerDispatcher::run() @ 0x134794b8 in /usr/bin/clickhouse
8. Poco::PooledThread::run() @ 0x13661cd0 in /usr/bin/clickhouse
9. Poco::ThreadImpl::runnableEntry(void*) @ 0x1365f664 in /usr/bin/clickhouse
10. start_thread @ 0x7624 in /usr/lib/aarch64-linux-gnu/libpthread-2.31.so
11. ? @ 0xd149c in /usr/lib/aarch64-linux-gnu/libc-2.31.so
 │
│ UNKNOWN_FUNCTION              │   46 │ UNKNOWN_FUNCTION              │     1 │ Unknown function Int64                                                                                                                                                                                                                                     │
│ UNKNOWN_IDENTIFIER            │   47 │ UNKNOWN_IDENTIFIER            │    19 │ Missing columns: '$symbol' while processing query: 'SELECT (intDiv(toUInt32(message_send_time_ts), 20) * 20) * 1000 AS t, process_name, symbol, avg(toInt64(colour)) FROM tsdb.traffic_lights WHERE (message_send_time_ts >= toDateTime(1685049068)) AND (message_send_time_ts <= toDateTime(1685070668)) AND ((`$symbol` = 'All') OR (symbol IN (`$symbol`))) AND (process_name IN (`$es_process_name`)) GROUP BY process_name, symbol, t ORDER BY process_name ASC, symbol ASC, t ASC', required columns: 'process_name' 'symbol' '$symbol' 'message_send_time_ts' 'colour', maybe you meant: ['process_name','symbol','symbol','message_send_time_ts','colour'] │
│ UNKNOWN_TABLE                 │   60 │ UNKNOWN_TABLE                 │    17 │ Table tsdb.yield_curve_pricer_test doesn't exist                                                                                                                                                                                                           │
└───────────────────────────────┴──────┴───────────────────────────────┴───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

**Comment by Slach** [comment]
ok. fix will prepare on 0.21.1

**Comment by Slach** [comment]
as a workaround, you could directly change 
```
kubectl set image deployment/clickhouse-operator metrics-exporter=altinity/metrics-exporter:0.21.1 --namespace=<your-clickhouse-operator-namespace>
```

**Comment by alex-zaitsev** [comment]
Fixed in 0.21.1
